### DISCUSSION TITLE
Display Per-category bbox AP50 in coco evaluation

### DISCUSSION THREAD

**Discussion by deepaksinghcv** [facebookresearch/detectron2#2683] [Q&A] [answered]
Hello,
The COCOEvaluator prints `Per-category bbox AP`, but is there a way to print `Per-category bbox AP50`.
I have converted PASCAL VOC annotations to COCO, and PASCAL VOC evaluates at AP50.

Any suggestions would be really helpful.
Thanks in advance.

**Comment by deepaksinghcv** [comment, marked as answer]
@ppwwyyxx Do you have any suggestions for this?

**Reply by ppwwyyxx** [reply]
https://github.com/facebookresearch/detectron2/issues/1877
